doc/introduction/introduction.rst

```diff
@@ -1,162 +1,71 @@
 Introduction
 ============
 
-.. role:: html(raw)
-   :format: html
-
-.. raw:: html
-
-    <style>
-    .figure img {
-      position: relative;
-      float: right;
-    }
-    .sphx-glr-thumbcontainer {
-      float: right;
-      box-shadow: none;
-      min-height: unset;
-      margin-top: 0;
-      margin-right: 20px;
-    }
-    .sphx-glr-thumbcontainer:hover {
-      box-shadow: none;
-    }
-    .sphx-glr-thumbcontainer .caption {
-      display: none;
-    }
-    .sphx-glr-thumbcontainer .figure {
-      width: 200px;
-    }
-    .sphx-glr-thumbcontainer img {
-      width: 100%;
-    }
-    </style>
-
 .. image:: /_static/NOON.png
     :align: left

```

Comment:
```suggestion
    :align: right
```
This it might look better right-aligned?
![image](https://user-images.githubusercontent.com/49409390/83158762-2b1bb080-a0d3-11ea-8197-65a356d01938.png)